### ISSUE TITLE
Option to remove tax bands

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine#154]
This can be done by setting the rate equal to the band below it, but a user suggested a feature to do this directly.

**Comment by nikhilwoodruff** [comment]
This raises the possibility of implementing a React component that edits an entire parameter scale, rather than individual parameters for each rate and threshold.

**Comment by MaxGhenis** [comment]
Yes - see https://github.com/PolicyEngine/policyengine/issues/189
